code/FormSchemaController.php

```diff
@@ -33,14 +30,6 @@ abstract class FormSchemaController extends AdminController
      */
     private ?FormSchema $schema = null;
 
-    protected function init()
-    {
-        $this->beforeExtending('onInit', function () {
-            $this->prepareTinyMce();
-        });
-        parent::init();
-    }
```

Comment:
Doesn't need to be deprecated since the method still exists in the parent class.